migration-guide/redhat-to-ibm.adoc

```diff
@@ -87,13 +87,13 @@ _Now that we have those files we can start to migrate our environment._
 
 === Step 2: Update Maven Configuration
 
-MAVEN_SITE_URL_BASE}[Apache Maven] is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information.
+{MAVEN_SITE_URL_BASE}[Apache Maven] is a software project management and comprehension tool. Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information.
```

Comment:
Do we even have this as a variable?

Reply:
Yep, it leads to https://maven.apache.org/